README.md

```diff
@@ -29,6 +29,8 @@ To install the bundle on a project, add the following lines to your composer.jso
 ]
 ```
 
+Due to a lack of Symfony Flex recipe to do so automatically. In your projects `/config/packages` folder, create `ise_web_security.yaml` and populate it with the yaml config detailed below. 
```

Comment:
I think it's worth creating a sample config file as well, `ise_web_security.yaml.dist` or similar so people have something to copy in or use as a reference.

Reply:
Good shout, I'll create one of those. I'll create a few example config files in [here](https://github.com/GoogleChromeLabs/IseWebSecurityBundle/wiki) too